components/molecules/table/__tests__/stories.js

```diff
@@ -67,3 +67,5 @@ export const formatter = () => <Table columns={columnsFormatter} rows={data} />
 export const actions = () => <Table columns={columnsActions} rows={data} />
 
 export const withPagination = () => <Table columns={columnsActions} rows={data} pagination />
+
+export const loading = () => <Table columns={columns} rows={data} loading />
```

Comment:
@majiiid-drykiss Give me a story with loading where the columns and rows are not defined yet

Reply:
Done